src/mcl.jl

```diff
@@ -92,7 +92,7 @@ function _mcl_inflate!(dest::Matrix{Float64}, src::Matrix{Complex128}, inflation
     src_norm = vecnorm(src)
     min_rel = -1E-3*src_norm
     min_img = 1E-3*src_norm
-    @inbounds for (i, el) in enumerate(src)
+    for (i, el) in enumerate(src)
```

Comment:
Keep `@inbounds` here, it makes a difference (even if ideally the fact that `enumerate` is used would allow skipping the checks). In general, no unrelated changes should be made in a PR.